file_formats/ie_formats/ini.htm

```diff
@@ -389,7 +389,7 @@
         <br />
 
         <strong>check_by_view_port = boolean_value</strong><br />
-        This attribute appears to be unimplemented.<br />
+        This attribute appears to be unimplemented. <strong>Note:</strong> There are plenty of uses of the <em>check_view_port</em> attribute which is most likely unimplemented as well.<br />
```

Comment:
So this something that Bubb didn't cover? Then please ask him for this final detail.

Reply:
Confirmed that _check_view_port_ is not supported by the engine at all.